### DISCUSSION TITLE
Show service container alias in run output

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#12505] [Ideas]
As reported by Kasper Hermansen:

> Service containers are great, the host names are bit weird in the output, i.e. the generated hostname, would be nice if it showed the alias more clearly.

**Comment by shykes** [comment]
FYI @mircubed @vito

**Comment by kjuulh** [comment]
Probably somewhere around here. But who am I to say =D

https://github.com/dagger/dagger/blob/b0c7547414b58e3cc03422fccb922b677ed56934/core/container.go#LL1289C1-L1289C56

**Comment by shykes** [comment]
@vito is this made moot by the new TUI?

**Comment by vito** [comment]
Not sure; hard to say what output the original issue is referencing, but I don't think the situation has changed with the new TUI, since it doesn't go out of its way to show service aliases.

We at least don't have those `service DDSKYFHAKOU` groups anymore! Now you just see a `start <command>` span, like this:

![image](https://github.com/dagger/dagger/assets/1880/48cbdc5f-ae65-4b1c-98ae-14b24a9a2919)
